python/audit/tests/test_bundle.py

```diff
@@ -0,0 +1,80 @@
+from datetime import datetime
+
+from audit.models import (
+    AUDIT_OPERATION_MAP,
+    AuditLog,
+    AuditLogOperationResult,
+    AuditLogOperationType,
+)
+from django.urls import reverse
+from rest_framework.response import Response
+
+from adcm.tests.base import BaseTestCase
+
+
+class TestBundle(BaseTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.audit_operation_upload_bundle = AUDIT_OPERATION_MAP["UploadBundle"]["POST"]
+        self.audit_operation_load_bundle = AUDIT_OPERATION_MAP["LoadBundle"]["POST"]
+
+    def test_upload_success(self):
+        self.upload_bundle()
+
+        log: AuditLog = AuditLog.objects.first()
+
+        assert not log.audit_object
```

Comment:
is this mean that we expect object is absent? in general it is debatable cause we set object_type, but not set an object

Reply:
If we have no object we don't create record in audit object table. Or we have to create audit object in any case ?